docs/topics/spiders.rst

```diff
@@ -181,9 +181,9 @@ scrapy.Spider
        scraped data and/or more URLs to follow. Other Requests callbacks have
        the same requirements as the :class:`Spider` class.
 
-       This method, as well as any other Request callback, must return an
-       iterable of :class:`~scrapy.Request` and/or :ref:`item objects
-       <topics-items>`.
+       This method, as well as any other Request callback, must return
+       :class:`~scrapy.Request` objects, :ref:`item objects <topics-items>`, or
```

Comment:
I'd change this to "a Request object, an item object [...]" (keeping proper formatting of course), as it can only return one object if not returning an iterable.

Reply:
Okay, I've made the changes.